modules/development/ssh.nix

```diff
@@ -30,6 +31,10 @@ in
     config = mkIf cfg.enable {
       services.openssh.enable = true;
       users.users.root.openssh.authorizedKeys.keys = authorizedKeys;
-      users.users.${config.ghaf.users.accounts.user}.openssh.authorizedKeys.keys = authorizedKeys;
+      users.users.${config.ghaf.users.accounts.user}.openssh.authorizedKeys = {
+        keys = authorizedKeys;
+        # Waypipe-ssh key is used here to create keys for ssh tcp tunneling for pulseaudio.
+        keyFiles = ["${pkgs.waypipe-ssh}/keys/waypipe-ssh.pub"];
```

Comment:
The setting from this line should be in targets/lenovo-x1-carbon.nix file

Reply:
Thanks, fixed.
Earlier i did not find a correct place for host in lenovo-x1-carbon.nix but now i did.